cmake/quickstart-runner.cmake

```diff
@@ -0,0 +1,92 @@
+# ~~~
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+# A CMake script to run the quickstart programs.
+#
+# This script runs the "quickstart" program for a given library. We use CMake as
+# a scripting language because we know CMake to be installed and usable for both
+# Windows, Linux, and macOS. At least while building with CMake.
```

Comment:
"At least while building with CMake." should be part of the previous sentence ... perhaps parenthetical.

Reply:
Thanks. Fixed both entries.